doc/manuals/admin/perf_tuning.md

```diff
@@ -22,6 +22,14 @@ to use WireTiger storage engine) implements per-document lock. Thus, the lock sy
 From a performance point of view, it is recommended to use MongoDB 3.0 with WireTiger, especially
 in update-intensive scenarios.
 
+In addition, take into account the following information from official MongoDB documentation, as it may have
+impact in performance:
+
+* Check that ulimit settings in your system are ok. MongoDB provides [the following recomendations](https://docs.mongodb.org/manual/reference/ulimit)
+  As described in that document, in RHEL/CentOS you have to create a /etc/security/limits.d/99-mongodb-nproc.conf file,
+  in order to set soft/hard process limit to at least 32000 (check details in the cited document).
+* [Disable Transparent Huge Pages (THP)](https://docs.mongodb.org/manual/tutorial/transparent-huge-pages/)
```

Comment:
Queda un poco árido esto, yo metería un poco de literatura en plan: "You should also disable Transparent Huge Pages to increment the performance as explain in..."